### PR TITLE
rewrite more ajax -> fetch

### PR DESCRIPTION
**Pull request by piuccio**
## What does this change?
- Breaking news
- Snaps in fronts
- Fixtures in tag pages
## What is the value of this and can you measure success?

27 more occurrences of `utils/ajax` and we can get rid of `reqwest`
## Does this affect other platforms - Amp, Apps, etc?

No
## Request for comment

@stephanfowler I've deleted an old test of yours

### CODE REVIEW

File: static/test/javascripts/spec/common/onward/breaking-news.spec.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
     $,
+    config,
+    fetchJson,
     storage,
-    ajax,
     defaults,
     Promise
 ) {
-    describe('Breaking news', function () {
+    fdescribe('Breaking news', function () {
```
Comment (gtrufitt) [inline static/test/javascripts/spec/common/onward/breaking-news.spec.js]: OOOOOOOH

Comment (gtrufitt) [inline static/test/javascripts/spec/common/onward/breaking-news.spec.js]: I reference the `fdescribe` of course ;)

File: static/test/javascripts/spec/common/onward/breaking-news.spec.js [hunk 2]
```diff
@@ -56,9 +57,9 @@ define([
                         return breakingNews();
                     }).then(function(result) {
                         // make sure the DOM has finished updating
-                        requestAnimationFrame(function () {
+                        setTimeout(function () {
```
Comment (gtrufitt) [inline static/test/javascripts/spec/common/onward/breaking-news.spec.js]: Are you basically skipping a frame here, yeah?

**Comment by gtrufitt** [comment]
Apart from the `fdescribe`, looks good, nice work! 👍

File: static/test/javascripts/spec/common/onward/breaking-news.spec.js (hunk 2 above)
Comment (piuccio) [inline static/test/javascripts/spec/common/onward/breaking-news.spec.js]: yup, the source code is doing a `setTimeout(() => promise.write(), 0)`, so the test has to wait for more than one animation frame

File: static/test/javascripts/spec/common/onward/breaking-news.spec.js (hunk 1 above)
Comment (piuccio) [inline static/test/javascripts/spec/common/onward/breaking-news.spec.js]: :facepalm:

**Pull request merged**